src/rpc/util.h

```diff
@@ -39,27 +41,42 @@ struct RPCArg {
         AMOUNT,        //!< Special type representing a floating point amount (can be either NUM or STR)
         STR_HEX,       //!< Special type that is a STR with only hex chars
     };
+
+    enum class Optional {
+        /** Required arg */
+        NO,
+        /**
+         * Optinal arg that is a named argument and has a default value of
```

Comment:
spelling optinal

Reply:
yea.. why was this ignored?

Reply:
> yea.. why was this ignored?

Just in order to able to merge the pr quickly (it was big and needed to be rebased a lot). The spelling could be fixed later.